src/components/forums/forum/TopicList.tsx

```diff
@@ -9,6 +9,7 @@ import {
   selectRepliesFromPosts,
   selectTopics,
   sortByVotes,
+  selectForumPosts
 } from "../../../utils/posts";
 import { ForumData } from "../../../utils/hooks";
 
```

Comment:
can we also add the same functionality to view the topic in the list, before it gets finalized?

Reply:
I think the reason why this is problematic because the topic that we show can't be clickable, so there's not that much point in having it. The tracking ticket design doc lists Create Topic as one of the actions that should be waited upon so I didn't implement it here

Reply:
should we just include the `confirmTransaction` call here then so the user knows to wait?

Reply:
In 01bf67d6c8165a025ed8b4f3046bdc97affcada8 I rigged the UI to show a spinner when the topic is added. It's ugly at the moment, but I don't really have any time to make it look better. @ana2201 could you also take a look at this to make the UI look less awkward? Probably what we want is the list of topics, with one spinner appended to the end if the `newTopicInFlight` variable is set to `true`.